docs/user-guide/images.md

```diff
@@ -69,6 +69,20 @@ images in the ECR registry.
 The kubelet will fetch and periodically refresh ECR credentials.  It needs the
 `ecr:GetAuthorizationToken` permission to do this.
 
+Requirements:
+
+- You must be using kubelet version `v1.2.0` or newer.  (e.g. run `/usr/bin/kubelet --version=true`).
+- Your nodes must be in the same region as the registry you are using
+- ECR must be offered in your region
+
+Troubleshooting:
+
+- Verify all requirements above.
+- Get $REGION (e.g. `us-west-2`) credentials on your workstation. SSH into the host and run Docker manually with those creds. Does it work?
+- Verify kubelet is running with `--cloud-provider=aws`.
+- Check kubelet logs (e.g. `journalctl -t kubelet`) for log lines like:
+  - `plugins.go:56] Registering credential provider: aws-ecr-key`
+  - `provider.go:91] Refreshing cache for provider: *aws_credentials.ecrProvider`
 
```

Comment:
I'm sure these troubleshooting steps are all accurate and great but I feel like they're quite terse. For each of these four things, can you dig up a docs link that will help them execute these steps, and be sure to specify the full command each time, not just the flags? e.g.

- Verify kubelet is running with `kubelet --cloud-provider=aws`.